### DISCUSSION TITLE
github.io 404 error

### DISCUSSION THREAD

**Discussion by paulvickers** [community/community#101397] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

 
![Screenshot 2024-01-22 at 11 33 22](https://github.com/community/community/assets/4840626/c8cd3041-1425-4afd-88ba-e88c89aa6809)

My github.io site is suddenly returning a 404 error but I have not changed anything. Is this an error?

**Comment by brenoepics** [comment]
Is your repository private? if it is fine, try to make it public.

**Reply by paulvickers** [reply]
Well, I never changed anything from when it did work. But, as the screenshot above shows, the Visibility menu already shows it as public.

**Reply by brenoepics** [reply]
GitHub Pages visibility is public yeah, but maybe the repository is private or try to check from which branch is GitHub generating the pages, if it was not deleted
![image](https://github.com/community/community/assets/59066707/803d953c-6b61-4b65-a4eb-7b2d0b8fec7d)

**Reply by paulvickers** [reply]
Weirdly, it doesn't show the Build and deployment options.

**Comment by paulvickers** [comment]
So, I went to the repo settings (which had always been private) and made it public and now it works. Concerned that I now have to make the repo itself public which I never had to before.

**Reply by seyLu** [reply, marked as answer]
#### Hi @paulvickers,

You prolly had a Github Pro account before or something equivalent and it just expired. Me, for example, have the GitHub Student Pack which includes a free Github Pro account for a year I think.

It's on GitHub's docs, the info about [GitHub's pricing plans](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-pro):
![image](https://github.com/community/community/assets/98249191/2d1045ca-cb47-4653-82c6-c8448303d28c)

**Reply by paulvickers** [reply]
Hmm, could be it. I have (or maybe had?) and academic licence. Need to look into that, thanks.
